index.html

```diff
@@ -78,142 +66,138 @@ <h4 class="list-heading">Skills</h4>
                     <li class="items">Gitlab</li>
                     <li class="items">Terminal</li>
                 </ul>
-
             </div>
         </section>
-
-        <div class="link-page">
-            <hr>
-
-            <ul class="icon">
-                <li><img src="images/github1.png" alt="github" class="icons"></li>
-                <li><img src="images/linkden.png" alt="linkden1" class="icons "><img src="images/linkden2.png"
-                        alt="linkden1" class="icons "></li>
-                <li><img src="images/1.png" alt="1" class="icons"></li>
-                <li><img src="images/twitter1.png" alt="twitter" class="icons"></li>
-                <li><img src="images/2.png" alt="2" class="icons"></li>
-            </ul>
-        </div>
-
-        <section class="middle-section">
-
-            <h1>MY RECENT WORK</h1>
-            <hr>
-            <img src="images/snapshot.png" alt="Snapshoot">
-            <h2>Multi-Post Stories </h2>
-            <p>A daily selection of privately personalized reads; no accounts or sign-ups required.
-                has been the industry's standard dummy text ever since the 1500s, when an unknown
-                printer took a standard dummy text.</p>
-            <ul class="programs">
-                <li class="lang">css &nbsp;|</li>
-                <li class="lang">html &nbsp;|</li>
-                <li class="lang">bootstrap |</li>
-                <li class="lang"> Ruby</li>
-            </ul>
-            <div>
-                <button class="button" type="button" onclick="">See Project</button>
+        <img class="hr-img" src="./Images/Vector1.png" alt="hr">
+        <section>
+            <div class="link-page">
+                <ul class="icon">
+                    <li>
+                        <img src="./images/Element-Let.png" alt="let" class="work-left">
+                    </li>
+                    <li><img src="./images/github1.png" alt="github" class="icons"></li>
+                    <li><img src="./images/linkden.png" alt="linkden1" class="icons "><img src="./images/linkden2.png"
+                            alt="linkden1" class="icons "></li>
+                    <li><img src="./images/1.png" alt="1" class="icons"></li>
+                    <li><img src="./images/twitter1.png" alt="twitter" class="icons"></li>
+                    <li><img src="./images/2.png" alt="2" class="icons"></li>
+                    <li>
+                        <img src="./images/Element-Right.png" alt="right" class="work-right">
+                    </li>
+                </ul>
```

Comment:
- [ ] Kindly look into these icons. They seem to have broken both at mobile and desktop versions.

### Desktop broken icons

![desktop-broken-icons](https://user-images.githubusercontent.com/26597432/188196324-bbb3fe48-1276-48bb-9fe8-072131732e70.png)

### Mobile broken icons

![mobile-broken-icons](https://user-images.githubusercontent.com/26597432/188196370-2586c286-3a1d-454e-8b69-c140803c7c17.png)

Reply:
![Screenshot 2022-09-02 at 18 09 06](https://user-images.githubusercontent.com/37203742/188204377-c737672d-26ec-4350-9b35-871d2c62cc07.png)